ci-operator/step-registry/openshift-qe/write-perfscale-results/openshift-qe-write-perfscale-results-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: openshift-qe-write-perfscale-results
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-write-perfscale-results-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: ocp-qe-perfscale-es
+    mount_path: /secret
+  - namespace: test-credentials # this entry injects the custom credential
```

Comment:
@qiliRedHat please let me know what the correct namespace and name are here. Doesn't look to be correct

Reply:
@paigerube14 I added the credential for sa-google-sheet here https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/openshift-qe/sa-google-sheet
  "secretsync/target-name": "sa-google-sheet",
  "secretsync/target-namespace": "test-credentials,ci"